package.json

```diff
@@ -35,6 +35,9 @@
     "@babel/preset-env": "^7.13.10",
     "@babel/preset-react": "^7.12.13",
     "@babel/runtime": "^7.13.10",
+    "@eslint/compat": "^1.1.1",
```

Comment:
These are required by the new flat config (you get the instruction to install them after running the migration).